lib/tasks/tests.rake

```diff
@@ -0,0 +1,5 @@
+desc "Run Karma tests"
+task :test do

```

Comment:
Let's rename this file `js_tests.rake` and the task as well to `js_tests`.

We can have a `test` task that does both rspec and karma.